src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.java

```diff
@@ -0,0 +1,61 @@
+package me.zeroeightsix.kami.module.modules.combat;
+
+import me.zero.alpine.listener.EventHandler;
+import me.zero.alpine.listener.Listener;
+import me.zeroeightsix.kami.module.Module;
+import me.zeroeightsix.kami.module.ModuleManager;
+import me.zeroeightsix.kami.setting.Setting;
+import me.zeroeightsix.kami.setting.Settings;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraftforge.event.entity.player.AttackEntityEvent;
+
+/**
+ * @author polymer
```

Comment:
What did polymer make?

Reply:
that's the joke lol
i talked to them about it and apparently it turned out identical to yours /shrug

Reply:
🤔  very suss